README.md

```diff
@@ -4,10 +4,14 @@
 ![CHECK-DIST](https://github.com/obvious/weaver/actions/workflows/check_dist.yml/badge.svg)
 [![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
 
-Weaver is a [GitHub Action] to transform the [Figma Token Studio] tokens to platform specific theme files
+Weaver is a [GitHub Action] that enables syncing of tokens from your design system Figma to code. It automates the process of updating your design system tokens in code each time a Figma token is updated. Hence, reducing developer intervention needed in maintaining tokens across platforms. 
+
+The action integrates with Figma [Token Studio](https://www.figma.com/community/plugin/843461159747178978/Tokens-Studio-for-Figma-(Figma-Tokens)) plugin to transform the token's JSON the plugin creates. The action supports Android and iOS platforms. 
 
 ## Usage
```

Comment:
@msasikanth I suppose we need a setup for Tokens Studio to link to Weaver. Don't we?

Reply:
Yeah, it will be useful to have, since we only support that JSON output.